webpack/assets/javascripts/react_app/redux/actions/notifications/index.js

```diff
@@ -18,10 +19,22 @@ import { DEFAULT_INTERVAL } from './constants';
 
 const interval = process.env.NOTIFICATIONS_POLLING || DEFAULT_INTERVAL;
 
-const getNotifications = url => get({ key: NOTIFICATIONS, url });
+const handleNotificationPollingError = (error, stopNotificationPolling) => {
+  if (error?.response?.status === 401) {
```

Comment:
This is a new feature that we don't use in Foreman (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)
And it's only available from March 2020, are we sure we want drop support for "old browsers"?

Reply:
thanks for bringing this up, I think that with babel we get the polyfills for older browser
cc @jeremylenz @sharvit thoughts on this?

Reply:
I think this is the supported browser list, [chrome > 26, firefox > 4...](https://github.com/zloirock/core-js#supported-engines)

Reply:
I think we need to add [https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining](https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining) to the babel plugins config in order for it to work

Reply:
I see that `@babel/plugin-proposal-optional-chaining` is installed for me in foreman so it should work, but I'm not sure where from it's installed